stack.yaml

```diff
@@ -0,0 +1,21 @@
+resolver: lts-7.1
+
+packages:
+- '.'
+- location:
```

Comment:
I tried this locally and at least with lts-7.3, I didn't need this if I added

- snap-loader-static-1.0.0.0

under extra-deps